### PR TITLE
Changes for version 2.1.0

### PR DESCRIPTION
**Pull request by abravosuse**
### PR creator: Description

Introducing changes for version 2.1.0

### CODE REVIEW

**Comment by abravosuse** [comment]
@tomschr this PR is not to be merged until the new version is out.

**Comment by tomschr** [comment]
@abravosuse Thanks Alberto! :+1: I haven't gone through it yet, but fixed a missing `</para>` tag in commit 4283531.

**Review by tomschr — CHANGES_REQUESTED** [review]
@abravosuse Thanks for the contribution! :+1: Have some ideas and suggestions.

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -83,8 +83,8 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
         <para>A simplified, reactive web UI targeting &sap; admins </para>
       </listitem>
       <listitem>
-        <para>Automated discovery of &sap; Systems, &sap; Instances, and
-          &hana; clusters</para>
+        <para>Automated discovery of &sap; Systems, &sap; Instances,
+          &hana; clusters and ASCS/ERS clusters</para>
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:87]: Yes, SAP customers should be familiar with what an ASCS/ERS cluster means.

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -1320,6 +1383,39 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
...
+                <imageobject>
+                  <imagedata fileref="trento-cleanup-host-details-view.png"
+                    width="70%" />
+                </imageobject>
+              </mediaobject>
+            </figure>
+      <para>When the user clicks on the cleanup button, a box will open asking for confirmation and,
+       if the user grants it, all the components discovered by the agent in that particular host, including the host
+       itself and other components that might depend on the ones running on the host, will be removed from the console.
+       For example, if the user cleans up the host where the Primary Application Server of an SAP System is deregistered,
```
[the remark this answers is not in the document]
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:1413]: The Primary Application Server is a unique name. That said, the sentence is wrong. It should say: "For example, if the user cleans up the host where the primary application server of an SAP System is registered," not "deregistered."

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -812,12 +813,27 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
+        </para>
+        <figure xml:id="fig-non-met-expectation-detail">
+          <title>Non-met expectation detail view</title>
+          <mediaobject>
+            <imageobject>
+              <imagedata width="70%" fileref="trento-web-nonmetexpectation.png"/>
+            </imageobject>
+          </mediaobject>
+        </figure>
+      </step>
```
[the remark this answers is not in the document]
Comment (tomschr) [reply trento/xml/article_sap_trento.xml]: I applied variant 1 in commit 5ce67d9b79

**Review by tomschr — APPROVED** [review]

**Pull request merged**